script/Arcueid/Day 5/05_00_ARC05_9.txt

```diff
@@ -37,23 +38,24 @@ Corpses swarm toward Arcueid.
 [sha:161949997ca865854873f5b9a7ed662f1e3382d5]{
 -- Page 563, Offset 10115.
 -- 極寒において真夏を連想させる行列。
-A procession reminiscent of midsummer in the midst of the extreme cold.
+A procession that calls to mind midsummer in the face of this extreme cold.
 }
 [sha:e07d9a158f2978fc0d058143be939c523301788d]{
 -- Page 563, Offset 10116.
 -- 炎天下、砂糖にたかる蟻そのものだ。
-They're like ants gathering sugar under the blazing sun.
+Like ants gathering sugar under the blazing sun.
 }
 [sha:14c8f35f0768b66d07f5f99ae67167202636eebb]{
 -- Page 563, Offset 10117.
 -- アルクェイドは殺される。
-Arcueid will be killed.
+All to kill Arcueid.
 }
 [sha:c651e8b6eb7fdff298baaf423158d900c8b7caf2]{
 -- Page 563, Offset 10118.
 -- あの死体どもに手足をもぎとられ、俺が／おまえがしたように、解体されて献上される。
-// Weirdness here with the split personality
-Those corpses will tear her limb from limb, just as I did, and offer up her dismembered body.
+Those corpses are here to tear her limb from limb――
+just as I, or maybe %{i}you%{/i} had――
```

Comment:
Honestly might go for `<I|Y o u >` because I feel like ruby excuses bullshit like this better